integrationTests/singleShard/block/common.go

```diff
@@ -6,4 +6,4 @@ import "time"
 const StepDelay = time.Second
 
 // P2pBootstrapDelay -
-const P2pBootstrapDelay = time.Second * 12
+const P2pBootstrapDelay = time.Second * 5
```

Comment:
integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

Reply:
done